src/widgets/modals/ModalContainer.js

```diff
@@ -95,8 +79,8 @@ export const ModalContainer = ({
     <View style={modalContainer}>
       <Modal
         visible={isVisible}
+        presentationStyle="fullScreen"
         animationType="slide"
-        fullScreen={fullScreen}
         transparent
```

Comment:
I had to remove `transparent` as I was getting a warning about the presentation style with transparent.

<img width="609" alt="Screen Shot 2020-11-21 at 9 12 48 AM" src="https://user-images.githubusercontent.com/9192912/99846356-08c6a380-2bdb-11eb-8749-2eb7fc6d02de.png">